### PR TITLE
Add lines prefixed with @ as a comment

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: splunk-conf.YAML-tmLanguage [hunk 1]
```diff
@@ -13,7 +13,7 @@ patterns:
   match: ^# DO NOT EDIT THIS FILE\!$
   name: invalid.illegal
 - comment: A comment
-  match: ^#.*$
+  match: ^[#|@].*$
```
Comment (shakeelmohamed) [inline splunk-conf.YAML-tmLanguage]: I don't think this is the right regex. It should either be `^[#@].*$` or `^(#|@).*$`, I'd prefer the former

Comment (ghost) [reply splunk-conf.YAML-tmLanguage]: Oh yeah, I did `^(#|@).*$`first but changed to square brackets and forgot to delete the pipe. Will send a commit in a moment

**Comment by ghost** [comment]
Alright, I replaced the previous commit, let me know if that is okay

**Comment by ghost** [comment]
Sorry, I just noticed that I might have messed up something by accident. I'll fix it in a second.

Edit: Now should be good.

**Comment by shakeelmohamed** [comment]
Looks good, thanks @jameskolce! Happy hacking 🎃 
Closes #26

**Comment by ghost** [comment]
Alright thanks!

**Pull request merged**